src/js/components/app.tsx

```diff
@@ -34,9 +34,9 @@ const RedirectToTeamProjectsView = () => <Redirect to="/projects" />;
 
 class App extends React.Component<Props, void> {
   public componentWillMount() {
-    const { isUserLoggedIn, loadTeamInformation } = this.props;
+    const { hasStoredUserCredentials, loadTeamInformation } = this.props;
 
-    if (isUserLoggedIn) {
+    if (hasStoredUserCredentials) {
       // TODO: team information is now being fetched in three places:
       // - Login View after getting user login information from Auth0
```

Comment:
Nope—there are still three ways that an authorized user can start using the app: after logging in, after signing up, or by opening Minard with a valid accessToken and email in their localStorage. If the login page is hosted by Auth0, it doesn't really change things.